src/NUnitFramework/framework/Constraints/ExactCountConstraint.cs

```diff
@@ -31,9 +31,19 @@ namespace NUnit.Framework.Constraints
     /// item in a collection, succeeding only if a specified
     /// number of items succeed.
     /// </summary>
-    public class ExactCountConstraint : PrefixConstraint
+    public class ExactCountConstraint : Constraint
     {
-        private int expectedCount;
+        private readonly int expectedCount;
+        private readonly IConstraint itemConstraint;
```

Comment:
Will do. On a side note, is that something you can put in the dotSettings files I see in the repo so that ReSharper warns people like me about the new convention out of the box?